include/irr/asset/ICPUVirtualTexture.h

```diff
@@ -287,7 +368,7 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
         core::address_allocator_traits<phys_pg_addr_alctr_t>::multi_free_addr(storage->tileAlctr, m_addrsArray->size(), m_addrsArray->data(), m_addrsArray->data());
 
         //free entries in page table
-        if (!base_t::free(_addr, _img))
+        if (!base_t::free(_addr))
```

Comment:
how do you protect against double free?

Only master should free the physical tiles, not the alias.